courses/free-courses-en.md

```diff
@@ -650,6 +650,7 @@
 * [Tailwind CSS Tutorial](https://youtube.com/playlist?list=PL4cUxeGkcC9gpXORlEHjc5bgnIi5HEGhw) - The Net Ninja
 * [Web Development By Doing: HTML / CSS From Scratch](https://www.udemy.com/course/web-development-learn-by-doing-html5-css3-from-scratch-introductory/) - Udemy
 * [What the Flexbox](https://flexbox.io) - Wesbos
+* [HTML for Beginner](https://www.w3schools.com/html/) - W3School
```

Comment:
Move this resource to line 641 to resolve the lint error

Reply:
I have moved the resource.